aws/policy/application-services.yaml

```
@@ -180,3 +180,18 @@ Statement:
       - states:UpdateStateMachine
     Resource:
       - arn:aws:states:us-east-1:{{ aws_account_id }}:*
+  - Sid: Cloudformation
+    Effect: Allow
+    Action:
+      - cloudformation:CreateChangeSet
+      - cloudformation:DescribeStacks
+      - cloudformation:GetStackPolicy
+      - cloudformation:DescribeStackEvents
+      - cloudformation:CreateStack
+      - cloudformation:GetTemplate
+      - cloudformation:DeleteStack
+      - cloudformation:UpdateStack
+      - cloudformation:DescribeChangeSet
+      - cloudformation:ListChangeSets
+      - cloudformation:ListStackResources
+    Resource: arn:aws:cloudformation:*:*:stack/*/*
```

```suggestion
    Resource: arn:aws:cloudformation:us-east-1:{{ aws_account_id }}:stack/ansible-test*
```
We generally keep CI policy restricted to us-east-1, with our account, and prefer to use a predictable resource name in tests.

You'll also need to add `cloudformation:ListStacks` to the`AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees` Sid, to allow the terminator to list stacks.  The rest of these permissions look good.

---

The requested change have been made